src/components/AddBook.jsx

```diff
@@ -38,8 +39,8 @@ const AddBook = () => {
   };
 
   return (
-    <div>
-      <h1>Add New Book</h1>
+    <div className="add-book-form">
+      <h1 className="title-add-book">Add New Book</h1>
```

Comment:
- [ ] Great job working on this milestone. We have a few issues to address before you can move on to the next milestone. As per the Zeplin design, the text `Add New Book` aligns with the form field for Book title. Currently that is not the case with your design. Kindly see screenshots below and fix this alignment issue.
- Zeplin Design
![image](https://github.com/iamsjunaid/Bookstore/assets/62849872/250a5e08-386f-4976-b9ed-5e1b67d852ef)

- Your Design
![image](https://github.com/iamsjunaid/Bookstore/assets/62849872/d3173be6-7cb6-4967-a74c-1e5f82c8d31f)